Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Clickhouse/DriverFee.hs

```diff
@@ -132,12 +136,13 @@ findAllByDate merchantId statuses mbFrom mbTo dayBasis mbCollBy = do
                       CH.+. CH.unsafeCoerceNum @(Maybe Int) @(Maybe Centesimal) driverFee.govtCharges
```

Comment:
don't we need to add `specialZoneAmount` to `totalAmount`?

Reply:
`totalAmount = openMarketAmount + specialZoneAmount`